Formula/dnsdist.rb

```diff
@@ -0,0 +1,30 @@
+class Dnsdist < Formula
+  desc "Highly DNS-, DoS- and abuse-aware loadbalancer"
+  homepage "https://www.dnsdist.org/"
+  url "https://downloads.powerdns.com/releases/dnsdist-1.2.0.tar.bz2"
+  sha256 "9885c9ee8ac7076aede586ea58d4642eb877e7b2d76c902254494e2a5a5faa78"
+
+  depends_on "pkg-config" => :build
+  depends_on "lua"
+  depends_on "libedit"
+  depends_on "boost"
+
+  def install
+    ENV.cxx11
+
+    ENV["SDKROOT"] = MacOS.sdk_path if MacOS.version == :sierra
```

Comment:
what error?

Reply:
comment added. Note that the pdns formula has the same line, this is not a coincidence.